CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * BraintreeShopperInsights (BETA)
   * Bug fix: remove internal code signing from framework
 * BraintreePayPal
+  * Make `userAuthenticationEmail` property optional for all App Switch flows
```

Comment:
We will want to merge the latest from main into here so this is under the correct section.